schemas/public/onpremises-kfd-v1alpha2.json

```
@@ -578,29 +600,29 @@
                 "gpg_key_id"
               ]
             },
-            "yum": {
+            "YUM": {
```

Why this is capital now? (just a curiosity)

---

this is a mistake, I replaced too much occurrences. Thanks for pointing it out.
```suggestion
            "yum": {
```